source/data/search_serial_in_large_CRL-output.txt

```diff
@@ -8,20 +8,21 @@ crl_signed_content:    (4, 8, 511) 508 Bytes
 
 *** The CRL lists 8 certificates.
 serial:  999183360   position: (155, 157, 191)
-serial:  1043934336   position: (266, 268, 302)
-serial:  1084184741   position: (414, 416, 450)
 serial:  1167825894   position: (192, 194, 228)
+serial:  1167830238   position: (229, 231, 265)
+serial:  1043934336   position: (266, 268, 302)
+serial:  1018259643   position: (303, 305, 339)
 serial:  1018438612   position: (340, 342, 376)
 serial:  1018438937   position: (377, 379, 413)
-serial:  1018259643   position: (303, 305, 339)
-serial:  1167830238   position: (229, 231, 265)
+serial:  1084184741   position: (414, 416, 450)
```

Comment:
Strange, how come that serials and position has changed?

Reply:
I think Python3 dictionaries are ordered, so if you add things to it when you go through them you get them in the order you added them in. I remember Python2 being random, I'm not 100% on this though. The serials are still the same, only the orders changed.

Reply:
Oh yes, here's something I found about it: "If you want the same behavior as earlier versions of Python 2.x in both 2.7 (which uses an arbitrary sort order instead) and 3.x (which refuses to sort dicts)," from this [answer](https://stackoverflow.com/a/25675708/6077951)